README.md

```diff
@@ -147,26 +147,28 @@ Once you've done that, restart the IDE.
 
 ## Run Locally
 
-* Build and run tests
+* Build and create jar:
 
 ```shell
-./gradlew clean build
```

Comment:
I would leave (maybe under different subtitle) this original approach since it is quite useful when debugging and understanding the whole process. I usually do localy like this:
1. ./gradlew clean build
2. docker run --rm --pull always -p 8000:8000 -d surrealdb/surrealdb:latest start --user root --pass root --log debug
3. ./gradlew bootRun to start the application or start application in debug mode